components/tabs/tab.js

```diff
@@ -30,13 +40,12 @@ class Tab extends TabMixin(LitElement) {
 
 	renderContent() {
 		const contentClasses = {
-			'd2l-tab-handler': true,
```

Comment:
This wasn't used and I didn't need it in my implementation. Can re-add later if it's found to be needed.